apps/hash-graph/hash_graph/Makefile.toml

```diff
@@ -28,6 +28,8 @@ args = ["Miri is disabled as unsafe code is forbidden"]
 
 [tasks.test-integration]
 private = false
+env_files = ["${REPO_ROOT}/.env"]
+env = { HASH_GRAPH_PG_DATABASE = "${HASH_GRAPH_PG_DEV_DATABASE}" }
 extend = "cargo"
 args = ["nextest", "run", "--cargo-profile", "${CARGO_MAKE_CARGO_PROFILE}", "@@split(CARGO_TEST_FLAGS, )", "--workspace", "--test", "integration", "--profile", "integration", "${@}"]
```

Comment:
Not sure where to comment this, but I got the following output from `cargo make test`:
```
[cargo-make] INFO - Execute Command: "cargo" "hack" "--workspace" "--feature-powerset" "nextest" "run" "--cargo-profile" "dev"
error: process didn't exit successfully: `/home/ahmad/.rustup/toolchains/nightly-2023-01-23-x86_64-unknown-linux-gnu/bin/cargo metadata --format-version=1` (exit status: 101)
--- stderr
error: could not find `Cargo.toml` in `/home/ahmad/dev/hash/hash/apps/hash-graph` or any parent directory
```

Reply:
This is the same issue as we had before, the command has to be run from the `hash_graph` subdirectory. This will be fixed with #1932

Reply:
Oh yes, i just realized. Thank you for clarifying. I also assume it will be fixed in that follow-up :+1: